loader/stage0/efi/Makefile

```diff
@@ -6,30 +6,53 @@
 #
 ################################################
 
-CSOURCES = stage0.c guest_setup.c efi_boot_param.c
-ASOURCES = stage0_entry.S
+CSOURCES = stage0.c efi_boot_param.c
 
 include $(PROJS)/rule.linux
 
-OBJS = $(OUTDIR)/stage0_entry.o \
-	$(OUTDIR)/stage0.o \
+OBJS =  $(OUTDIR)/stage0.o \
 	$(OUTDIR)/stage0_lib.o \
 	$(OUTDIR)/efi_boot_param.o \
-	$(OUTDIR)/guest_setup.o \
 	$(LIB_DIR)/elf_ld.o \
 	$(LIB_DIR)/elf32_ld.o \
 	$(LIB_DIR)/elf64_ld.o \
+	$(LIB_DIR)/efi_services.o \
+	$(LIB_DIR)/efi_conf_table.o \
+	$(LIB_DIR)/efi_mp.o \
+	$(LIB_DIR)/efi_ap.o \
 	$(LIB_DIR)/util.o \
-	$(LIB_DIR)/string.o \
-	$(LIB_DIR)/sprintf.o
+	$(LIB_DIR)/string.o
 
 ifneq (, $(findstring -DLIB_PCI, $(EVMM_CMPL_FLAGS)))
 OBJS += $(LIB_DIR)/pci.o
 endif
 
 ifneq (, $(findstring -DLIB_PRINT, $(EVMM_CMPL_FLAGS)))
+OBJS += $(LIB_DIR)/sprintf.o
 OBJS += $(LIB_DIR)/print.o
 OBJS += $(LIB_DIR)/serial.o
 endif
```

Comment:
move these to the rule.link?

Reply:
I don't want to impact other stage0, maybe do this later.

Reply:
ignore this comment. we can do it later when the difference accumulates.